experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -1,14 +1,341 @@
+            </tr>
+            <tr>
+              <th>Secondary Outcomes</th>
+              <td colspan="3">
+                {% for outcome, url in secondary_outcome_links %}
+                  <a href="{{ url }}" target="_blank" rel="noopener noreferrer">{{ outcome|format_to_title }}</a>
+                  {% if not forloop.last %},{% endif %}
+                {% empty %}
+                  <span style="color: red; ">Not set</span>
+                {% endfor %}
+              </td>
+            </tr>
+            <tr>
+              <th>Team projects</th>
+              <td>
+                {{ experiment.projects.all|join:"<br>"|format_not_set|safe }}
+              </td>
+              <th>Subscribers</th>
+              <td>
+                {{ experiment.subscribers.all|join:"<br>"|format_not_set|safe }}
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <!-- Risk Mitigation Questions Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Risk Mitigation Questions</h4>
+      </div>
+      <div class="card-body">
+        <table class="table table-striped">
+          <tbody>
+            <tr>
+              <th colspan=3>{{ RISK_QUESTIONS.BRAND }}</th>
+              <td style="{% if experiment.risk_brand|yesno == 'Yes' %}color: red;
+                         font-weight: bold;
+                         {% endif %}">{{ experiment.risk_brand|yesno|format_not_set|safe }}</td>
+            </tr>
+            <tr>
+              <th colspan="3">{{ RISK_QUESTIONS.MESSAGE }} Message Consult</th>
+              <td style="{% if experiment.risk_message|yesno == 'Yes' %}color: red;
+                         font-weight: bold;
+                         {% endif %}">{{ experiment.risk_message|yesno|format_not_set|safe }}</td>
+            </tr>
+            <tr>
+              <th colspan="3">{{ RISK_QUESTIONS.REVENUE }}</th>
+              <td style="{% if experiment.risk_revenue|yesno == 'Yes' %}color: red;
+                         font-weight: bold;
+                         {% endif %}">{{ experiment.risk_revenue|yesno|format_not_set|safe }}</td>
+            </tr>
+            <tr>
+              <th colspan="3">{{ RISK_QUESTIONS.PARTNER }}</th>
+              <td style="{% if experiment.risk_partner_related|yesno == 'Yes' %}color: red;
+                         font-weight: bold;
+                         {% endif %}">{{ experiment.risk_partner_related|yesno|format_not_set|safe }}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <!-- Audience Card -->
+    <div class="card mb-3">
+      <div class="card-header">
+        <h4>Audience</h4>
+      </div>
+      <div class="card-body">
+        <table class="table table-striped">
+          <tbody>
+            <tr>
+              <th>Channel</th>
+              <td style="text-transform: capitalize;">{{ experiment.channel|default:"No Channel" }}</td>
```

Comment:
Oh yeah I think as @brennie pointed out you should be able to do 

```
experiment.channel.title
```

instead of using the text transform?